apps/dashboard/src/components/wallets/ConnectWalletPlayground/ThemeButton.tsx

```diff
@@ -1,6 +1,7 @@
-import { Tooltip } from "@chakra-ui/react";
+import { Button } from "@/components/ui/button";
+import { ToolTipLabel } from "@/components/ui/tooltip";
+import { cn } from "@/lib/utils";
 import { useTrack } from "hooks/analytics/useTrack";
-import { Button } from "tw-components";
 
 export function ThemeButton(props: {
```

Comment:
This is only used in landing page btw

Reply:
yeah... anyway

Reply:
ah it's removed now. so i will remove this file too